csp_tools/aws/dcgmi_diag.sh

```diff
@@ -19,4 +19,4 @@
 
 # This is a Data Center GPU Manager container. This command will run GPU diagnostics.
 # This script should not be called manually. It should only be called by cluster_validation.sh
-srun --container-image=nvcr.io/nvidia/cloud-native/dcgm:2.3.5-1-ubi8 bash -c "dcgmi diag -r 3"
+srun --container-image=dcgm.sqsh bash -c "dcgmi diag -r 3"
```

Comment:
I think we would still want this as the NGC image link. I'm not sure why pulling from NGC wasn't working for you, but using the NGC registry image requires the least work.

Reply:
sounds good, reversed this